### PR TITLE
Overview about sharing files and folders in OneDrive or SharePoint.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

File: api-reference/v1.0/api/driveitem-invite.md [hunk 1]
```diff
@@ -60,21 +67,22 @@ In the request body, provide a JSON object with the following parameters.
 | recipients       | Collection([DriveRecipient][]) | A collection of recipients who will receive access and the sharing invitation.
 | message          | String                         | A plain text formatted message that is included in the sharing invitation. Maximum length 2000 characters.
 | requireSignIn    | Boolean                        | Specifies whether the recipient of the invitation is required to sign-in to view the shared item.
-| sendInvitation   | Boolean                        | If true, a [sharing link][] is sent to the recipient. Otherwise, a permission is granted directly without sending a notification.
+| sendInvitation   | Boolean                        | If true, a sharing link is sent to each recipient. Otherwise, a permission is granted directly without sending a notification.
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/driveitem-invite.md:70]: @kevklam , in line 70, is a sharingLink resource sent to each recipient along with the email notification?
My guess it's just a web URL, not really a sharingLink resource?

File: api-reference/beta/api/driveitem-createlink.md [hunk 2]
```diff
@@ -42,7 +42,13 @@ POST /sites/{siteId}/drive/items/{itemId}/createLink
...
 ```
 
-### Request body
+## Request headers
+| Header       | Value |
+|:---------------|:--------|
+| Authorization  | Bearer {token}. Required.  |
+| Content-Type  | application/json  |
+
+## Request body
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/driveitem-createlink.md:51]: @kevklam , per our earlier chat, the beta CSDL includes 2 additional parameters for createLink - message and recipients.
Can you please confirm if that's intended and therefore the parameters should be documented? Thanks.

**Comment by jasonjoh** [comment]
Please disregard the earlier message from msftbot. Your pull request was incorrectly flagged.

**Comment by angelgolfer-ms** [comment]
I will come back to work on this PR in the May/June timeframe, after the higher priority deliverables for Build are taken care of.

**Pull request closed without merging**